packages/@aws-cdk/aws-backup/README.md

```diff
@@ -132,8 +132,10 @@ const vault = new backup.BackupVault(this, 'Vault', {
 })
 ```
 
-Use the `blockRecoveryPointDeletion` property to add statements to the vault access policy that
-prevents recovery point deletions in your vault:
+Alternativately statements can be added to the vault policy using `addToAccessPolicy()`.
+
+Use the `blockRecoveryPointDeletion` property or `blockRecoveryPointDeletion()` method to add
```

Comment:
Is there any additional use case that the late-bound `blockRecoveryPointDelection()` exposes? If not, I prefer we point to the `blockRecoveryPointDeletion` property as the way to add this feature to the access policy statement and make the implementation details private.

Reply:
Allows to do this:
```ts
const plan = new BackupPlan(this, 'BackupPlan');
plan.backupVault.blockRecoveryPointDeletion();
```

instead of
```ts
const plan = new BackupPlan(this, 'BackupPlan');
plan.backupVault.addToAccessPolicy(/* block statement here */);

// OR

const vault = new BackupVault(this, 'BackupVault', { blockRecoveryPointDeletion: true });
const plan = new BackupPlan(this, 'BackupPlan', { backupVault: vault });
```

your call!

Reply:
Cool, I can see how that is useful